source/tyr/tests/integration/v1_test.py

```diff
@@ -197,3 +209,151 @@ def test_authorization_methods(create_5_users, create_instance, create_api):
         '/v1/users/{}/authorizations'.format(user_id), data=json.dumps(auth), content_type='application/json'
     )
     assert len(resp_delete['user']['authorizations']) == 0
+
+
+def test_autocomplete_parameters(create_autocomplete_parameters):
+    check_v1_response('autocomplete_parameters')
+    check_v1_response(
+        'autocomplete_parameters', 'autocomplete_parameters/{}'.format(create_autocomplete_parameters)
+    )
+
+
+def test_autocomplete_parameters_methods():
+    resp_get = api_get('/v1/autocomplete_parameters')
+    assert 'autocomplete_parameters' in resp_get
+    initial_num = len(resp_get['autocomplete_parameters'])
+    assert initial_num == 0
+
+    autocomplete_data = {
+        'street': 'OSM',
+        'name': 'france',
+        'address': 'BANO',
+        'admin': 'COSMOGONY',
+        'poi': 'OSM',
+    }
+    resp_post, status_post = api_post(
+        '/v1/autocomplete_parameters',
+        data=json.dumps(autocomplete_data),
+        content_type='application/json',
+        check=False,
+    )
+    assert status_post == 201
```

Comment:
minor: you may remove that and change the check in `api_post` to test `201` and not 200 (we may be surprised in other tests though :smile: )
Same on delete for 204.

Reply:
Changing it will surely break some other tests.
Also, the status checked in "api_delete" is 200. Meaning that some other v0 endpoints return 20 when deletion is OK?